src/compiler/parser.rs

```diff
@@ -856,20 +885,26 @@ impl ParserType {
         self.create_node_array(list, list_pos, None, Some(comma_start >= 0))
     }
 
+    fn create_missing_list(&self) -> NodeArray {
+        let mut list = self.create_node_array(vec![], self.get_node_pos(), None, None);
+        list.is_missing_list = true;
```

Comment:
The Typescript version models `MissingList` as a sub-type of `NodeArray`, but that seems like overkill/a lot of wiring-up (to turn `NodeArray` into an enum) so I just made all `NodeArray`'s have an `is_missing_list: bool` field